store/src/java/com/zimbra/cs/imap/MemcachedImapCache.java

```diff
@@ -106,7 +107,10 @@ public Object serialize(ImapFolder folder) throws ServiceException {
             } catch (Exception e) {
                 throw ServiceException.FAILURE("Failed to serialize ImapFolder", e);
             } finally {
-                Closeables.closeQuietly(oout);
+                try {
```

Comment:
can use apache commons-io IOUtils.closeQuietly instead.

Reply:
Or follow the recommendations in the deprecation statement for Closeables.closeQuietly and use "try with resources".  See http://grepcode.com/file/repo1.maven.org/maven2/com.google.guava/guava/15.0/com/google/common/io/Closeables.java?av=f